CHANGELOG.md

```diff
@@ -1,3 +1,99 @@
+## 1.3.4
+
+ABQ 1.3.4 is patch release with feature preview of remote persisted storage.
+
+A remote storage source can be used to synchronize and offload ABQ manifests and
+test results usually stored on local disk.
+The remote storage source is used to download offloaded manifests and test results
+on-demand.
```

Comment:
Could we clarify somewhere that `offload` in this context means uploading it to remote storage and deleting it from local storage?